modules/modal/react/lib/ModalContent.tsx

```diff
@@ -192,8 +196,9 @@ const ModalContent = ({
 
   React.useEffect(() => {
     const siblings = [...((container.children as any) as HTMLElement[])].filter(
-      el => el !== modalRef.current!.parentElement
+      el => el.tagName.toLowerCase() !== 'script' && el !== modalRef.current!.parentElement
```

Comment:
I've noticed script tags are hidden by `aria-hidden`, but what accessibility problems does this cause?

Reply:
I dont think it causes any problems. Its something that I noticed. But having it on the script does not bring any value either?

Reply:
Sometimes in older versions of iOS when using explore by touch with VoiceOver running, scripts included within the <body> tag would incorrectly be announced, so as a belt and braces approach aria-hidden="true" would be added to the <script> element.

This is since been resolved in the later iterations - there's no impact here for a11y.